pyiron_atomistics/vasp/base.py

```diff
@@ -672,8 +707,13 @@ def from_directory(self, directory):
             else:
                 raise ValueError("Unable to import job because structure not present")
             self.structure = structure
-            # Always set the sorted_indices to the original order when importing from jobs
-            self.sorted_indices = np.arange(len(self.structure), dtype=int)
+            # Always set the _idx_pyiron_to_user to the original order (unsorted) when importing from jobs
+            try:
+                self._idx_pyiron_to_user = np.arange(len(self.structure), dtype=int)
+            except:
```

Comment:
I'm struggling a bit to see when this exception will/won't be thrown. As soon as someone's running this version of the code, every Vasp job instance should have the read-only property `self._idx_pyiron_to_user`, and thus the try will fail with an `AttributeError`. Is this correct? If it is *always* failing, what is it that we're actually trying to accomplish with the try/except and can we do it in a more meaningful way? It's possible this bit is just fine and I'm misunderstanding things, but I might need some hand-holding here.


```suggestion
            except AttributeError:
```
It's good form to be as targeted as possible when catching exceptions

Reply:
To be completely honest, this is only here because it failed tests.

I would prefer to have it not exist, but one of the tests fails if I remove this. Something about trying to access an attribute of an old job that doesn't exist throws an error that fails tests. So if at some point if we want to remove backwards compatibility and instead get users to re-generate the hdf5s so that they are valid with this implementation, we can get rid of this try; except.